### ISSUE TITLE
Program page: Tool-tip explanations of %Met "on track" highlights

### ISSUE THREAD

**Issue by jennyhavah** [mercycorps/TolaActivity#1007]
**Note: This ticket is dependent on ticket #1006**

# Issue
Users are experiencing confusion about our "on track" metrics and how they relate (or don't) to the %Met calculations displayed in the results table. As a result, we're getting reports of calculation errors when the queries are, in fact, behaving as expected.

The proposed solutions use a combination of visual highlights (covered in #1006) and context-specific tool-tips (covered in this ticket) to clearly communicate these relationships.

**Affects:** The results table, %Met results, _when they are highlighted in yellow or green._ 

# Acceptance criteria
Clicking on a highlighted %Met result produces a tool-tip, like so:
![image](https://user-images.githubusercontent.com/33670923/50497071-7961e700-09e9-11e9-9ff2-69632cb1d121.png)

## When the %Met is highlighted in green
- [x] 1. The title of the tool tip is **ON TRACK** in bold all caps
- [x] 2. The message reads as follows, with the appropriate value replacing XX%. (Note that the first sentence is highlighted in bold text, and the last sentence is in italics.)

**The actual value is XX% of the target value.** An indicator is on track if the result is no less than 85% of the target and no more than 115% of the target.

_Remember to consider your direction of change when thinking about whether the indicator is on track._

## When the %Met is highlighted in yellow
- [x] 1. The title of the tool tip is **NOT ON TRACK** in bold all caps
- [x] 2. The message reads as follows, with the appropriate value replacing XX%. (Note that the first sentence is highlighted in bold text, and the last sentence is in italics.)

**The actual value is XX% of the target value.** An indicator is on track if the result is no less than 85% of the target and no more than 115% of the target.

_Remember to consider your direction of change when thinking about whether the indicator is on track._

**Comment by jennyhavah** [comment]
Looks great. The padding on the bottom doesn't bother me -- I think it actually helps set the tooltip apart a bit more from the background.

As long as it's trivial to implement, I wonder if a subtle shadow effect behind the tool tip would make it easier to visually differentiate the tool tip from a busy background? (Low priority.)

**Comment by axoplasm** [comment]
Weird but true: Bootstrap box shadows are kind of an all-or-nothing business. If we give the tooltips box shadows, all kinds of other gets shadows (and inset shadows) — form elements, etc.

**Comment by jennyhavah** [comment]
> Weird but true: Bootstrap box shadows are kind of an all-or-nothing business. If we give the tooltips box shadows, all kinds of other gets shadows (and inset shadows) — form elements, etc.

@axoplasm How fun. (Bleargh.)

**Comment by RickieT** [comment]
Looks like this is functoning as intended based on acceptance criteria. On multiple results, the proper colors/text were shown On track/Not on track based on % values.

**Comment by RickieT** [comment]
Closing as fixed/deployed
